tokio-util/tests/foo.rs

```diff
@@ -0,0 +1,33 @@
+use bytes::Bytes;
+use tokio::io::{AsyncReadExt, Result};
+use tokio_util::io::StreamReader;
+#[tokio::main]
+async fn main() -> std::io::Result<()> {
```

Comment:
AFAICT, this is a duplicate of the test `stream_reader::test_stream_reader`, except that this one won't be run with the rest of the tests since it uses `tokio::main` rather than `tokio::test`?

Am I missing something here? Should this be removed, or moved to the examples dir or something?

Reply:
I'm not quite sure what happened here.